pipeline_plugins/components/utils/sites/open/utils.py

```diff
@@ -255,3 +255,27 @@ def get_biz_ip_from_frontend(
             data.outputs.ex_data = _("IP 为空，请确认是否输入IP,请检查IP格式是否正确：{}".format(ip_str))
         return False, ip_list
     return True, ip_list
+
+
+def get_biz_ip_from_frontend_hybrid(executor, ip_str, biz_cc_id):
+    """
+    处理前端ip,云区域:ip 混合输入的情况，最终提取出来的格式为:
+    [
+        {
+            "bk_cloud_id": 0,
+            "ip": 127.0.0.1
+        }
+    ]
+    """
+    # 先提取所有跨业务的ip
+    plat_ip = [match.group() for match in plat_ip_reg.finditer(ip_str)]
+    plat_ip_list = [{"ip": _ip.split(":")[1], "bk_cloud_id": int(_ip.split(":")[0])} for _ip in plat_ip]
```

Comment:
plat_ip = [match.group().split(":") for match in plat_ip_reg.finditer(ip_str)]
[{"ip": _ip, "bk_cloud_id": int(_bkcloud))} for _ip, _bkcloud in plat_ip]